apps/transport/lib/transport/comments_checker.ex

```diff
@@ -127,19 +126,25 @@ defmodule Transport.CommentsChecker do
         comment_timestamp(comment)
 
       [c | comments] ->
-        DgDate.latest_dg_datetime(comment_timestamp(c), comments_latest_timestamp(comments))
+        latest_datetime(comment_timestamp(c), comments_latest_timestamp(comments))
+    end
+  end
+
+  defp latest_datetime(%DateTime{} = date1, %DateTime{} = date2) do
+    case DateTime.compare(date1, date2) do
+      :lt -> date2
+      _ -> date1
     end
```

Comment:
```suggestion
    Enum.max([date1, date2], DateTime)
```

C'est ma petite [PR](https://github.com/elixir-lang/elixir/pull/11564) de doc Elixir ;)

Reply:
Grandiose, je prends ! 💖